src/AutoRest.CSharp/Mgmt/Output/ResourceCollection.cs

```diff
@@ -138,8 +138,6 @@ protected override IEnumerable<ContextualParameterMapping> EnsureExtraContextual
             var suppressListException = Configuration.MgmtConfiguration.ListException.Contains(RequestPath)
                 || Configuration.MgmtConfiguration.MgmtDebug.SuppressListException;
             var getAllOperation = ClientOperations.Where(operation => operation.Name == "GetAll").OrderBy(operation => ReferenceSegments(operation).Count()).FirstOrDefault();
-            if (!suppressListException && getAllOperation == null)
```

Comment:
There wasn't much value for this check since the service team always add a suppression for it.
So, we decided to remove it.

Reply:
keep this with TODO

Reply:
@m-nash Had another talk with Shanghai team, we still think it's not worth to keep this check and add suppression every time it breaks, it causes much more support effort than what gains. So, we would like to remove it for good.